PactNet.sln

```diff
@@ -1,264 +1,232 @@
-﻿
```

Comment:
The file still seems to have line ending problems. I think `.sln` files have to have `\r\n` line endings if I remember correctly, so if you've made this commit on a non-Windows machine that may be the problem.

Perhaps add a `.gitattributes` file to force `.sln` to have `CRLF` line endings?

Reply:
I'm not seeing any line ending issues at this point, there were a number of errors that had to fixed with a rebase to get the EventImporter sample that are now addressed. The diff is showing just the expected added lines for me, let me know if you still see the line endings showing up wrong.